src/test/java/org/rulez/demokracia/pdengine/UnimplementedTests.java

```diff
@@ -14,19 +14,9 @@
 @TestedOperation("Unimplemented")
 @TestedBehaviour("Unimplemented")
 public class UnimplementedTests extends CreatedDefaultChoice {
-	
-	@Test
-	public void the_empty_assurances_can_be_obtained() {
-		CastVote castVote = new CastVote("test_user_in_ws_context", new ArrayList<>(), new ArrayList<>());
-		List<String> assurances = castVote.getAssurances();
-		assertTrue(assurances.isEmpty());
-	}
-	
-	@Test
-	public void the_not_empty_assurances_can_be_obtained() {
-		List<String> assurances = new ArrayList<String> ();
-		assurances.add("TestAssurance");
-		CastVote castVote = new CastVote("test_user_in_ws_context", new ArrayList<>(), assurances);
-		assertTrue(castVote.getAssurances().contains("TestAssurance"));
-	}
+
+//	@Test
```

Comment:
uh-oh

Reply:
What should I do?
Create a new part of the code with the unimplemented exception?

Reply:
If something is not part of the code, then it should not be in the code.
Do not leave comments behind.